From abd49fa183b074e208fcea226855b568fceae3ee Mon Sep 17 00:00:00 2001
From: Han Gao <gaohan@iscas.ac.cn>
Date: Tue, 17 Dec 2024 23:50:58 +0800
Subject: [PATCH 310/414] dts: eic770x: megrez: add force 1.8Ghz

milkv ensures that all chips can pass the 0.9V1.8Ghz test.

Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts b/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
index 3fbee7bee395..7839bd28d00f 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
@@ -879,6 +879,11 @@ &dev_llc_d0{
 	apply_npu_high_freq;
 };
 
+&d0_clock {
+	status = "okay";
+	force-1_8ghz;
+};
+
 &d0_cpu_opp_table {
         opp-1500000000 {
                 opp-hz = /bits/ 64 <CLK_FREQ_1500M>;
-- 
2.47.0

